NextCenturyCorporation/itm-evaluation-dashboard
# Itm 467

When the text-based scenarios are completed, we send all of the user choices to the TA1 servers to obtain alignment scores. 

Note:
- Going through the text-based scenarios should create 4 entries in the `userScenarioResults` collection (one for each scenario).
- Each of these documents should have a separate high and low alignment
- You must pull a fresh copy from [adept's server repo](https://gitlab.com/itm-ta1-adept-shared/adept_server) since the change to make this functional just got merged in last night 
- For future scenarios, this will be a lot cleaner, the `problemProbes.json` is needed to map MRE decisions to proper probe choices. Another function inside `TextBasedScenarios.jsx` named `mapAnswers` will not be necessary for the dry run. 

To test:
Run through the text-based scenarios and ensure your answers are properly recorded in the `userScenarioResults` collection of mongo and are accompanied by a high and low alignment score. Be sure to have the TA1 servers running locally and that you are using the dev `.env` variables and not prod. You will most-likey have to rebuild your dashboard docker container.
